src/views/product-tutorials-view/server.ts

```diff
@@ -25,7 +26,10 @@ export interface ProductTutorialsPageProps {
   product: ProductTutorialsPageProduct
 }
 
-type ProductTutorialsLayout = CollectionLayout
+type ProductTutorialsLayout = Pick<
```

Comment:
`ProductTutorialsLayout` previously mirrored `CollectionLayout`, but now it does not, as the former needs `headings` while the latter has no use for them.